setup.cfg

```diff
@@ -0,0 +1,8 @@
+[isort]
```

Comment:
I'm loathe to introduce a new config file. Is there is a reason for defining these in `setup.cfg` as well as in `.pre-commit.yaml`?

Reply:
I definitely understand your sentiment. I don't like config files either, especially not root-level ones. The reason for this file is that CLIs and IDEs don't read config from `.pre-commit-config.yaml`. So if you enabled format on save in your editor or you use `flake8` or `mypy` or `isort` directly instead of through `pre-commit run mypy|isort|...`, they would run with their default settings and make changes that would be reverted by `pre-commit`. E.g. try `flake8 src` in a terminal. It complains about 100 of lines too long because it doesn't know about the line length set in `.pre-commit-config.yaml`.

Like I said, I understand your sentiment so if you think this is insufficient reason, I'll of course delete the file.

Reply:
It does work the other way round btw. Settings here will also be picked up when running these tools through pre-commit. So we could remove the config from `.pre-commit-config.yaml` to keep a SSOT.

Reply:
Yeah, that would be preferable!